### PR TITLE
crc: Add version 1.34.0-4.9.0

### PR DESCRIPTION
**Pull request by maanur**

### CODE REVIEW

File: bucket/crc.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        "architecture": {
+            "64bit": {
+                "url": "https://mirror.openshift.com/pub/openshift-v4/clients/crc/$version/crc-windows-amd64.zip"
+            }
+        },
+        "hash": {
+            "url": "$baseurl/sha256sum.txt",
+            "regex": "($sha256)\\s+crc-windows-amd64.zip"
+        },
+        "extract_dir": "crc-windows-$version-amd64"
```
Comment (Ash258) [inline bucket/crc.json:27]:
```suggestion
        }
```

File: bucket/crc.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://mirror.openshift.com/pub/openshift-v4/clients/crc/$version/crc-windows-amd64.zip"
+            }
+        },
+        "hash": {
+            "url": "$baseurl/sha256sum.txt",
+            "regex": "($sha256)\\s+crc-windows-amd64.zip"
```
Comment (Ash258) [inline bucket/crc.json:25]:
```suggestion
            "url": "$baseurl/sha256sum.txt"
```

File: bucket/crc.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    },
+    "extract_dir": "crc-windows-1.8.0-amd64",
+    "bin": "crc.exe",
+    "checkver": {
+        "github": "https://github.com/code-ready/crc"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://mirror.openshift.com/pub/openshift-v4/clients/crc/$version/crc-windows-amd64.zip"
```
Comment (Ash258) [inline bucket/crc.json:20]:
```suggestion

                "url": "https://mirror.openshift.com/pub/openshift-v4/clients/crc/$version/crc-windows-amd64.zip",
                "extract_dir": "crc-windows-$version-amd64"
```

File: bucket/crc.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    "description": "Red Hat CodeReady Containers is a tool that manages a local OpenShift 4.x cluster optimized for testing and development purposes.",
+    "version": "1.8.0",
+    "license": "Apache-2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://mirror.openshift.com/pub/openshift-v4/clients/crc/1.8.0/crc-windows-amd64.zip",
+            "hash": "a6e87ef46699d5d8da1e3960818a57b53f329ea9a190d40ec025371f23547183"
+        }
+    },
+    "extract_dir": "crc-windows-1.8.0-amd64",
```
Comment (Ash258) [inline bucket/crc.json:12]:
```suggestion
```

File: bucket/crc.json [hunk 5]
```diff
@@ -0,0 +1,29 @@
+{
+    "homepage": "https://code-ready.github.io/crc/",
+    "description": "Red Hat CodeReady Containers is a tool that manages a local OpenShift 4.x cluster optimized for testing and development purposes.",
+    "version": "1.8.0",
+    "license": "Apache-2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://mirror.openshift.com/pub/openshift-v4/clients/crc/1.8.0/crc-windows-amd64.zip",
+            "hash": "a6e87ef46699d5d8da1e3960818a57b53f329ea9a190d40ec025371f23547183"
```
Comment (Ash258) [inline bucket/crc.json:9]:
```suggestion
            "hash": "a6e87ef46699d5d8da1e3960818a57b53f329ea9a190d40ec025371f23547183",
            "extract_dir": "crc-windows-1.8.0-amd64"
```

File: bucket/crc.json [hunk 6]
```diff
@@ -0,0 +1,29 @@
+{
+    "homepage": "https://code-ready.github.io/crc/",
+    "description": "Red Hat CodeReady Containers is a tool that manages a local OpenShift 4.x cluster optimized for testing and development purposes.",
+    "version": "1.8.0",
```
Comment (Ash258) [inline bucket/crc.json:4]:
```suggestion
    "version": "1.8.0",
    "description": "Manages a local OpenShift 4.x cluster optimized for testing and development purposes.",
    "homepage": "https://code-ready.github.io/crc/",
```

File: bucket/crc.json [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://mirror.openshift.com/pub/openshift-v4/clients/crc/$version/crc-windows-amd64.zip",
+                "extract_dir": "crc-windows-$version-amd64",
+                "hash": {
+                    "url": "$baseurl/sha256sum.txt"
+                }
+            }
+        }
```
Comment (Ash258) [inline bucket/crc.json:27]:
```suggestion
                "extract_dir": "crc-windows-$version-amd64"
            }
        },
        "hash": {
            "url": "$baseurl/sha256sum.txt"
        }
```

**Review by Ash258 — DISMISSED** [review]

File: bucket/crc.json (hunk 7 above)
Comment (maanur) [reply bucket/crc.json:27]: Please explain the requested changes

Comment (Ash258) [reply bucket/crc.json:27]: What You do not understand?

This hash file is generic. It should be outside of architecture.

Comment (maanur) [reply bucket/crc.json:27]: Sincerely grateful for clarification!

**Comment by maanur** [comment]
Sorry, I can't figure out, what changes are requested now... @Ash258

**Comment by Ash258** [comment]
None. I just not gonna review 2GB something related to Red Hat. Thats why `please-review` labe.

**Comment by rashil2000** [comment]
Hey @maanur!

Can you update this to the latest version?

**Comment by issaclin32** [comment]
I have updated the manifest by myself because the PR is pretty outdated.

NOTES:
- about `crc_hyperv_4.9.0.crcbundle`: I have compared the SHA-256 values of this file in 2 configurations
(1) Intalling with the MSI installer
(2) Extract the file using *7-zip*, than concatenate using `copy /b`
The SHA-256 values are both `C7852E69EAB8457DC706406A6BA1C4C38D796212E9409501E9013667D72CD310`

**Review by rashil2000 — APPROVED** [review]

**Pull request merged**